cli-feed-v3.json

```diff
@@ -157,6 +157,17 @@
       "sha2": "FE046167D756F020D3B727106FD74C8A47E15D22EF1A05B1671A98DB03D074C4",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
+    "2.3.2": {
+      "Microsoft.NET.Sdk.Functions": "1.0.14",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.33/Azure.Functions.Cli.win-x86.2.0.1-beta.33.zip",
+      "nodeVersion": "8.11.1",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10210",
+      "projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10210",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.0-beta-10205.zip",
+      "sha2": "DEB64C4E7E4E9132F19A090A0F4FC05B8CEF930637B7351A8182AC0AECD809F7",
+      "FUNCTIONS_EXTENSION_VERSION": "beta"

```

Comment:
Add the following 2 properties
```
"requiredRuntime": ".NET Core",
"minimumRuntimeVersion": "2.1"
```

Reply:
As per our latest conversation with VS team, we will switch back to no-runtime once they start enforcing these properties

Reply:
I added the 2 properties